src/app/pages/chooseDoctor/chooseDoctor.component.scss

```diff
@@ -0,0 +1,82 @@
+@import '../../../assets/styles/colors';
```

Comment:
Tworzyłem komponent metodą crtr-c crtl-v, więc wkleiłem też tego cssa, ale nie wiem czy on cokolwiek robi

Reply:
jak nie uzywasz zadnych zmiennych z niego to nic nie robi